aws/src/integration/java/org/apache/iceberg/aws/glue/GlueTestBase.java

```diff
@@ -79,18 +79,18 @@ public static void beforeClass() {
     S3FileIO fileIO = new S3FileIO(clientFactory::s3);
     glueCatalog = new GlueCatalog();
     glueCatalog.initialize(catalogName, testBucketPath, new AwsProperties(), glue,
-            LockManagers.defaultLockManager(), fileIO);
+            LockManagers.defaultLockManager(), fileIO, ImmutableMap.of());
```

Comment:
Use this [initialize()](https://github.com/apache/iceberg/blob/b8a796effdca02108b2af005f977cbeb0d9cd718/aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java#L160) in the test ensures all fields are initialized. Same to the other related changes in the integration tests.